frontend/src/components/adjudications/AdjudicationTableRow.jsx

```diff
@@ -8,18 +8,25 @@ import IconButton from "@material-ui/core/IconButton";
 import EditIcon from "@material-ui/icons/EditOutlined";
 import DoneIcon from "@material-ui/icons/DoneAllTwoTone";
 import RevertIcon from "@material-ui/icons/NotInterestedOutlined";
+import LensIcon from '@material-ui/icons/Lens';
 import { Link } from 'react-router-dom';
-import { updateAdjudications, getAdjudicationByPerformanceAndJudge } from '../../api/AdjudicationAPI';
+import { updateAdjudications, getAdjudicationByPerformanceAndJudge, getJudgesWhoCompletedAdjudication } from '../../api/AdjudicationAPI';
 
 class AdjudicationTableRow extends React.Component {
-  state = {adjudication: {}, isEditMode: false, technicalMark: {}, artisticMark: {} };
+  state = {adjudication: {}, isEditMode: false, technicalMark: {}, artisticMark: {}, isAdmin: true, judges: {}}; //hardcoded admin value
 
   componentDidMount() {
     getAdjudicationByPerformanceAndJudge(this.props.id, 1) //hradcoded tablet_id
     .then(({data}) => {
       this.setState({ adjudication: data, technicalMark: data.technicalMark, artisticMark: data.artisticMark});
+    }); 
+    getJudgesWhoCompletedAdjudication(parseInt(this.props.id))
+    .then(({data}) => {
+      this.setState({judges: data});
```

Comment:
does `data` contain all the http boilerplate? like status code and headers and all that? 

by directly associating the api response with our state we might get some unwanted behaviours with future development

Reply:
Data is just an object containing the tablet_ids of the judges who completed that adjudication. I'm pretty sure it only contains integers.